### ISSUE TITLE
Change loaders to webpack 5 assets mangment

### ISSUE THREAD

**Issue by ghost** [WeAreAthlon/frontend-webpack-boilerplate#32]
**Is your feature request related to a problem? Please describe.**

NO

***What is the motivation / use case for changing the behavior?**

If webpack 5 have this system by default, i think it must be used

**Describe the solution you'd like**

Use the assets mangment https://webpack.js.org/guides/asset-management/

**Describe alternatives you've considered**

**Additional context**

```js
{
  test: /\.(png|gif|jpe?g|svg)$/i,
  type: 'asset',
  generator: {
    filename: 'images/[name].[hash:6].[ext][query]',
  },
  parser: {
    dataUrlCondition: {
      maxSize: environment.limits.images,
    },
  },
},
{
  test: /\.(eot|ttf|woff|woff2)$/,
  type: 'asset',
  generator: {
    filename: 'fonts/[name].[hash:6].[ext][query]',
  },
  parser: {
    dataUrlCondition: {
      maxSize: environment.limits.fonts,
    },
  },
},
```

**Comment by francosalcedo** [comment]
Thank you very much for your suggestion, this served as a solution to load fonts. In the latest versions of webpack the boilerplate generated fonts with javascript content inside "export default" fonts / ... "".

Thanks

**Comment by pnikolov** [comment]
The newly released [Release v5.10.0](https://github.com/WeAreAthlon/frontend-webpack-boilerplate/releases/tag/v5.10.0) is using the `webpack 5` assets management tools and is not more using `url-loader` and `file-loaders`. 
The [demo page](https://weareathlon.github.io/frontend-webpack-boilerplate/) is updated to verify the images and fonts loading.
